src/selectors/patient.js

```diff
@@ -50,7 +50,6 @@ export const selectPatientModalOpen = ({ patient }) => {
 
 export const selectCanEditPatient = ({ patient }) => {
   const { currentPatient } = patient;
-  const { thisStoresPatient } = currentPatient ?? {};
-
-  return thisStoresPatient ?? true;
+  const { isEditable = true } = currentPatient ?? {};
```

Comment:
possibly better to have this as false as the default

Reply:
Good point. 6d37591.